make-release.sh

```diff
@@ -118,7 +118,7 @@ releaseCheServer() {
 }
 
 releaseDevworkspaceGenerator() {
-    invokeAction eclipse-che/che-devfile-registry "Release Che Devworkspace Generator" "67742638" "version=${CHE_VERSION}"
+    invokeAction devfile/devworkspace-generator "Release Che Devworkspace Generator" "102323522" "version=${CHE_VERSION}"
```

Comment:
do we know if the bot that will trigger this action has permissions to trigger it from that repository?

Reply:
I've added write permissions for `che-bot`

Reply:
I've added write permissions to che-incubator-bot